package.json

```diff
@@ -56,15 +56,19 @@
 		"nest:start": "nest start server",
 		"nest:start:dev": "nest start server --watch",
 		"nest:start:debug": "nest start server --debug --watch",
-		"nest:start:prod": "node dist/apps/server/main",
+		"nest:start:prod": "node dist/apps/server/apps/server.app",
```

Comment:
apps/server/apps/server.app we should keep in mind that we can reduce the path apps/server in future if featherJS is completly removed, otherwise its read funny with doppel apps/server :P